src/proto/parameters.proto

```diff
@@ -435,3 +438,15 @@ message PossessionTrackerConfig
         (bounds).max_double_value = 10.0
     ];
 }
+
+message NetworkConfig
+{
+    // The referee interface
+    required string referee_interface = 1 [default = "lo"];
+
+    // The robot status interface
+    required string robot_status_interface = 2 [default = "lo"];
```

Comment:
nit: 
1. I think a name like `robot_interface` or `robot_communication_interface` is more suiting since this is used for everything coming and going to the robot.
2. Could you make this the first field in the proto message so it gets rendered as the first element in Thunderscope? Ideally the order of the fields here matches the print statements in the terminal.

Reply:
updated